app/authentication/login/login.component.e2e-spec.ts

```diff
@@ -0,0 +1,40 @@
+import { browser, by, element } from 'protractor'
+
+describe('Login', () => {
+
+  beforeEach(() => {
+    browser.get('/login');
+  });
+
+  it('should have an input and submit button', () => {
+
+    let allInputElements = element.all(by.css('md-card md-card-content form input'));
+    let inputElement = element(by.css('md-card md-card-content form input'));
+    let submitButton = element(by.css('md-card md-card-content form button'));
+    let loginCardTitle = element(by.css('md-card md-card-title h3'));
+
+    browser.wait(function () {
+      return inputElement.isPresent();
+    }, 60000);
+
+    expect(inputElement.isPresent()).toEqual(true);
+    expect(submitButton.isPresent()).toEqual(true);
+    expect(element(by.css('md-card md-card-title h3')).getText()).toEqual('Geben Sie Ihre Daten ein, um sich einzuloggen');
+
+    //send inputs
+    allInputElements.first().sendKeys('admin@hipapp.de');
+    allInputElements.last().sendKeys('Hipcms@123');
```

Comment:
Even in Test code: do not hardcode credentials.
We are working on an *open source* project. So everyone can look in our code and have now our login data to login as admin!

Better:
first create a user on the sign up page and then login with these credentials of the newly created user.